lib/utils.py

```diff
@@ -376,3 +376,37 @@ def str_to_bytes(s):
     if PY2:
         return s
     return s.encode('utf-8')
+
+
+def parse_tag_line(line):
+    tags_str = line.split(':', 1)[1].strip()
+    return [tag.strip() for tag in tags_str.split(',')]
+
+
+def find_tags(filename):
+    """ Extract tags from a first comment in the file.
+    """
+    # TODO: Support multiline comments. See exclude_tests() in
+    # lib/server.py.
+    if filename.endswith('.lua') or filename.endswith('.sql'):
+        singleline_comment = '--'
+    elif filename.endswith('.py'):
+        singleline_comment = '#'
+    else:
+        return []
+
+    tags = []
+    with open(filename, 'r') as f:
+        for line in f:
+            line = line.rstrip('\n')
+            if line.startswith('#!'):
+                pass
+            elif line == '':
+                pass
```

Comment:
So, it would like the following:

```
#!/usr/bin/env tarantool

    -- tags: lalala
    --
    -- Blah blah.
```

Or:

```
#!/usr/bin/env tarantool

    -- Blah blah.
    --
    -- tags: lalala
```

It would be quite strange to indent a top level comment this way. Since we accept only first comment (precisely: first non-empty block except maybe shebang, when it is a comment), it must be at the top level.

Another possibility is the 'empty' line that contains only whitespaces. Trailing whitespaces are highly discouraged in programming, even `git diff` colors them as red.

So, back to your question: yep, it is technically possible, but has no sense.